app/Console/Kernel.php

```diff
@@ -32,6 +32,6 @@ class Kernel extends ConsoleKernel
      */
     protected function schedule(Schedule $schedule)
     {
-        //
+        $schedule->command('queue:work --sleep=3 --tries=3')->everyMinute();
```

Comment:
I dunno. I feel like we should be able to tell them to add `schedule:run` to a cron and it's done. If they don't want it, they can use `sync` and it'll run *inline*. Easier that way?